action/tasks/devices/gpio.yml

```diff
@@ -0,0 +1,11 @@
+name: device-gpio
+required-by: [chroot]
+refers: target
+echo: true
+timeout: 10
+fail-fast: true
+disabled: true
+commands:
+  - "modprobe gpio-mockup gpio_mockup_ranges=${{ranges}}"
+vars:
+  ranges: "0,32"
```

Comment:
The var should probably be uppercase. Also: do we need to specify the default values for the variables? I think these should be empty in the device tasks.

Reply:
If the device variable is left empty by the user, the default value won't be overridden. I left variables lowercase because I thought this will be usefull to use variable names in yaml code that will replace current syntax, but I think now that I will do it other way. So I will change them to uppercase.